### DISCUSSION TITLE
SPH WOOD Screw Example with MUMPS

### DISCUSSION THREAD

**Discussion by MarkusSchulten** [OpenRadioss/OpenRadioss#2045] [Q&A]
Hi, is it general possible to solve SPH cases like the
Wood Screw Example (SPH)

https://altairuniversity.com/learning-library/radioss-sample-of-a-screwing-process/

with the implicit Solver Mumps? Maybe such a process is more efficient because of a larger time step?
Do you have hints for the parameters of the 0001.rad file?
I have used the parameters of the springback-example with minor changes, but I don´t get convergence with small time steps.
See the attached file, please.
[WOOD_SCREW_00_0001.rad.txt](https://github.com/OpenRadioss/OpenRadioss/files/13294381/WOOD_SCREW_00_0001.rad.txt)

Kindly regards
Markus

**Comment by MarianBulla** [comment]
Hi Markus, 
thank you very much for your interest in OpenRadioss. 
Unfortunately, the SPH method is only available in the explicit part, because of the "flying" particles it is quite difficult to find a solution for implicite methods. 
If you want to go with implicite, you can maybe remove the SPH section of that model and modifiy it, so you will see the damage before full failure of the material. 
Would that work for you ?
How long does the explicite solution take for you, using this example ?

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by MarkusSchulten** [comment]
Hi Marian,
thank you for your answer.
I am at 6days in Double Precision and 4days with Single Precision after a few minutes.
I would use the Wood Screw tutorial only as a test case for an experiment with large plastic strains and wanted to see the potential for avoiding a remeshing operation with SPH.
Best regards
Markus
